### DISCUSSION TITLE
Rectal administration of dosage forms like suppositories in PK-Sim

### DISCUSSION THREAD

**Discussion by Mwilaputa** [Open-Systems-Pharmacology/Forum#1139] [Questions & Problems] [answered]
I want to develop a PBPK model of the drug (in form of a suppository) that is administered via rectum but am getting stuck how to proceed on "Administration type" when I select "User Defined" under "Administration protocols" building block. Any help on how to proceed is highly appreciated.

**Comment by StephanSchaller** [comment, marked as answer]
Like this?
![image](https://user-images.githubusercontent.com/25477643/154839549-fee229c5-327c-4672-b862-758ebb813c35.png)

**Comment by Slow-Walker66** [comment]
there is another way, you could divide rectal administration into two part, one is IV, the other is p.o, like the below picture.
![image](https://user-images.githubusercontent.com/87760070/186053095-f0bfff73-ea42-4d93-ab8b-8f9d2bfbc0c1.png)
